documentation/inventory_files.md

```diff
@@ -468,6 +625,102 @@ plugins:
         message: Enter your Vault password
 ```
 
+### Switch transports between command
```

Comment:
```suggestion
### Switch transports between commands
```

Reply:
This section seems a bit out of place in the examples section. Maybe move it below the plugins section?